### ISSUE TITLE
Youtube TODO

### ISSUE THREAD

**Issue by markjamesbutler** [guardian/frontend#14635]
- [x] Separate re-usable hosted video components
- [x] fix AMP pages e.g. [http://localhost:9000/science/grrlscientist/2012/aug/07/3?amp](http://localhost:9000/science/grrlscientist/2012/aug/07/3?amp) to use amp-youtube
- [x] Multi videos per page
- [x]  Ophan Tracking
- [x] GA Tracking 
- [x] Poster image (dotcom pickup?)
- [x] Customise play button (dotcom pickup?)
- [x] Embeddable Guardian YouTube atom (Tracking too)
- [x] Testing cross device
- [x] Test Amp
- [x] Test Instant Articles
- [x] Test Apple news
- [x] Test Ads off platform
- [x] QA Testing
  Blocked by Tools:
- [x] Other video (main media, standalone pages...)
- [x] caption text
- [x] end slate

Nice to have:
- [x] Test overriden Poster image

**Comment by paperboyo** [comment]
Possible consideration for Poster image point above: https://github.com/guardian/frontend/pull/15193#issuecomment-265465338 (not blocking)

**Comment by akash1810** [comment]
🎉 All items complete! 🎉 

![img](https://media.giphy.com/media/AL0XsYU0pkFTq/giphy.gif)
